### PR TITLE
Create Officers Announcement

### PR DESCRIPTION
**Pull request by LEQUI000**

### CODE REVIEW

**Comment by CurtisGreen** [comment]
image

I think this part should be have some formatting changes to make it easier to read like line breaks, or more bolded headers, or bulleted list, etc. 
Kind of all blends in together rn

**Comment by LEQUI000** [comment]
> image
> 
> I think this part should be have some formatting changes to make it easier to read like line breaks, or more bolded headers, or bulleted list, etc. Kind of all blends in together rn

I believe this looks slightly better now?

image

**Comment by LEQUI000** [comment]
I need one more cover photo

File: _posts/data-advocates-for-stronger-funding-dart-board-sept-2024.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+FOR IMMEDIATE RELEASE
+
+**Dallas Area Transit Alliance Advocates for Stronger Transit Funding at DART Board Meeting**  
+
+DALLAS, TEXAS – On September 24, 2024, the Dallas Area Transit Alliance (DATA) filled the DART Board of Directors meeting with passionate advocates supporting a 5.5% increase in the operating budget for FY2025, as originally proposed by DART staff.
+
+XXX, a resident of XXX and member of DATA, addressed the board: XXX”
+
+XX continued, "XXX"
```
Comment (LEQUI000) [inline _posts/data-advocates-for-stronger-funding-dart-board-sept-2024.md]: Still need an anecdote from board meeting

**Review by CurtisGreen — APPROVED** [review]

**Pull request merged**